### PR TITLE
feat(cleanup): remove defaultprops

### PR DESCRIPTION
**Pull request by romain-guillot-symphony**

### CODE REVIEW

File: src/components/layout/dynamic-panel/index.js [hunk 1]
```diff
@@ -79,14 +79,14 @@ const ADDITIONAL_ICON_WIDTH = 60;
...
-    closeDebouncePerdiod,
+    tabs = null,
+    onChange = null,
+    activeTab = null,
+    onRemove = null,
+    renderMethod = 'single',
+    hasAddButton = true,
+    onCreate = null,
+    closeDebouncePerdiod = 500
   } = props;
```
Comment (StepankaBarotova) [inline src/components/layout/dynamic-panel/index.js:90]: A typo in `closeDebouncePerdiod`

**Review by StepankaBarotova — COMMENTED** [review]
LGTM!

**Comment by JulienBlacas** [comment]
@romain-guillot-symphony Any update about this PR ?

**Comment by romain-guillot-symphony** [comment]
> @romain-guillot-symphony Any update about this PR ?

Updated the PR, ready to merged :)

File: src/components/layout/box/index.js [hunk 2]
```diff
@@ -82,16 +105,3 @@ Box.propTypes = {
   my: PropTypes.string,
   type: PropTypes.oneOf([BOX_TYPES.PRIMARY, BOX_TYPES.SECONDARY, BOX_TYPES.FLAT]),
 };
-Box.defaultProps = {
-  display: 'flex',
-  direction: 'flex-column',
```
Comment (JulienBlacas) [inline src/components/layout/box/index.js:87]: @romain-guillot-symphony Direction should have a default value in the Box function parameters ?

Comment (romain-guillot-symphony) [reply src/components/layout/box/index.js:87]: `direction` was not used, it is a function defined bellow

**Review by JulienBlacas — APPROVED** [review]
LGTM !

**Pull request merged**